Public/Src/Sandbox/MacOs/Sandbox/Src/BuildXLSandbox.cpp

```diff
@@ -130,6 +130,8 @@ void BuildXLSandbox::InitializePolicyStructures()
         .mpo_vnode_check_write            = Listeners::mpo_vnode_check_write,
 
         .mpo_vnode_check_readlink         = Listeners::mpo_vnode_check_readlink,
+
+        .mpo_vnode_check_clone            = Listeners::mpo_vnode_check_clone,
```

Comment:
will add some test for this

@Kr1sso, if you send me the `csh` sourcing business because of which you had to implement `mpo_vnode_check_write`, I'll add a test for that too.

Reply:
Sourcing a file that uses a redirect operator in a csh script did not get registered originally, setup:

execute.sh
```
#!/bin/csh
source experiment.csh
```

experiment.csh
```
ls -lah ~/ > contents.txt
```